### PR TITLE
Clean-up Pipeline Service RBACs

### PR DESCRIPTION
**Pull request by Roming22**

### CODE REVIEW

**Review by sbose78 — APPROVED** [review]
/lgtm

File: components/pipeline-service/staging/base/team-support-rbac.yaml [hunk 1]
```diff
@@ -1,57 +0,0 @@
----
-kind: ClusterRole
```
Comment (gabemontero) [inline components/pipeline-service/staging/base/team-support-rbac.yaml:2]: @sayan-biswas refresh my memory - does this ClusterRole play a part in how we were getting the permissions we needed to use results to get logs, etc. for users on stage

I know the results specific resources are not here, but did we need the other items from here ?

Comment (gabemontero) [reply components/pipeline-service/staging/base/team-support-rbac.yaml:2]: yeah still need clarification form @sayan-biswas to see if https://issues.redhat.com/browse/PLNSRVCE-1381 was inadvertently getting some of the permissions it needs from the ClusterRole you have moved

I suspect the answer is "no" but I am not 100% certain, even after your helpful clarifications

Comment (gabemontero) [reply components/pipeline-service/staging/base/team-support-rbac.yaml:2]: bump @sayan-biswas ... especially in light of the discrepancy you are seeing in being able to use the CLI in staging but not in prod

**Pull request merged**